### ISSUE TITLE
No Paginate Option Results in Unpredictable Behavior

### ISSUE THREAD

**Issue by mdbenjam** [CircleCI-Public/aws-parameter-store-orb#16]
### Orb version

1.0.0

### What happened

My Parameter Store had more than ten values defined. Using this orb pulled only ten of them. My build succeeded, but was missing several parameters I actually needed.

### Expected behavior

Seems like this orb should pull all of the parameters, rather than stopping on the first page, which I think can be done by removing the `--no-paginate` from this line: https://github.com/CircleCI-Public/aws-parameter-store-orb/blob/269c941abea4802c90b9d2474f9b7dd2a9c387bf/src/commands/load.yml#L18.

If that behavior is not desired for some reason, it seems like having more than 10 parameters should cause the build to fail, so that builds don't pass with incorrect configuration.

**Comment by coelhorafaela** [comment]
I had a problem with this parameter too, but I took a specific parameter with the filter:
```
steps:
- aws-parameter-store/load:
          filter: "Key=Name,Values=/Dev/Parameter/Name"
- checkout
```
and it did not return, but when I ran this script on aws cli without the `--no-paginate`, the parameter was loaded.

**Comment by 0b01001110** [comment]
Hi, can I try this issue?

**Comment by neelrr1** [comment]
Yes, I think the `--no-cli-pager` option should be used instead.

See: https://github.com/aws/aws-cli/issues/2449
